### ISSUE TITLE
Investigate behaviour when naming tools via `Add tool dialogue`

### ISSUE THREAD

**Issue by denis-yuen** [ga4gh/dockstore#581]
### Expected behaviour

You should be able to register (as an example) all of quay.io/jeltje/radia, quay.io/jeltje/radia/radia, quay.io/jeltje/radia/x in any order (but with no duplicates).

We should also improve documentation in GUI during this process.

### Actual behaviour

It seems like the first tool registered comes without a toolname even if specified.

FYI @Jeltje

**Comment by agduncan94** [comment]
@denis-yuen I tried this with my own tool and it doesn't seem to be an issue anymore.

**Comment by denis-yuen** [comment]
@agduncan94  Yeah, this may have been fixed. If you can't duplicate, I think this code has changed enough that I think this can just be closed
